orchestration/dap_orchestration/solids.py

```diff
@@ -0,0 +1,122 @@
+
+@configured(extract_records)
+def hles_extract_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_extract_config(config, "org.broadinstitute.monster.dap.hles.HLESurveyExtractionPipeline")
+
+
+@configured(extract_records)
+def cslb_extract_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_extract_config(config,"org.broadinstitute.monster.dap.cslb.CslbExtractionPipeline")
+
+
+@configured(extract_records)
+def env_extract_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_extract_config(config, "org.broadinstitute.monster.dap.environment.EnvironmentExtractionPipeline")
+
+
+@solid(
+    required_resource_keys={"beam_runner"},
+    config_schema={
+        "output_prefix": String,
+        "target_class": String,
+    }
+)
+def transform_records(context: AbstractComputeExecutionContext, input_prefix: str) -> str:
+    """
+    :return: Returns the path to the transformation output json files.
+    """
+    arg_dict = {
+        "inputPrefix": input_prefix,
+        "outputPrefix": context.solid_config["output_prefix"],
+        "target_class": context.solid_config["target_class"],
+        "scala_project": "dog-aging-hles-transformation"
+    }
+    context.resources.beam_runner.run(arg_dict)
+    # todo: resolve mypy error: Returning Any from function declared to return "str"
+    return context.solid_config["output_prefix"]
+
+def _build_transform_config(config: dict[str, str], target_class: str) -> dict[str, str]:
+    return {
+        "output_prefix": config["output_prefix"],
+        "target_class": target_class
+    }
+
+
+# todo: how do I specify the input_values (output_prefix) for a configured function?
+@configured(transform_records)
+def hles_transform_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_transform_config(config, "org.broadinstitute.monster.dap.hles.HLESurveyTransformationPipeline")
+
+@configured(transform_records)
+def cslb_transform_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_transform_config(config, "org.broadinstitute.monster.dap.cslb.CslbTransformationPipeline")
+
+@configured(transform_records)
+def env_transform_records(config: dict[str, str]) -> dict[str, str]:
+    return _build_transform_config(config, "org.broadinstitute.monster.dap.environment.EnvironmentTransformationPipeline")
+
+
+@solid(
+    config_schema={
+        "working_dir": String,
+    }
+)
+def write_outfiles(context: AbstractComputeExecutionContext, input_prefix: str) -> str:
+    """
+    :return: Returns the path to the tsv outfiles.
+    """
+    os.mkdir("tsv_output")
+    subprocess.run(
+        ["python", "hack/convert-output-to-tsv.py", input_prefix, context.solid_config["working_dir"]+"/tsv_output", "--debug"],
+        check=True,
+        cwd=context.solid_config["working_dir"]
+    )
+    # todo: resolve mypy error: Returning Any from function declared to return "str"
+    return context.solid_config["working_dir"]+"/tsv_output"
```

Comment:
⛏️ Prefer f-strings, like so:
```suggestion
    return f"{context.solid_config["working_dir"]}/tsv_output"
```